### PR TITLE
Apply one line per sentence style to md files

### PR DESCRIPTION
**Pull request by ramchale**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

Apply one line per sentence style to md files. 

As this is the desired standard going forward, applying this to existing files will make it easier to adopt.

### CODE REVIEW

File: docs/book/v2/inflector.md [hunk 1]
```diff
@@ -88,20 +78,18 @@ $inflector = new Laminas\Filter\Inflector('#foo/#bar.#sfx', array(), null, '#');
...
-Additionally, you may wish to have a class member for your class that you can
-use to keep the inflector target updated &mdash; without needing to directly update
-the target each time (thus saving on method calls). `setTargetReference()`
-allows you to do this:
+Additionally, you may wish to have a class member for your class that you can use to keep the inflector target updated &mdash; without needing to directly update the target each time (thus saving on method calls).
+`setTargetReference()` allows you to do this:
 
-```php
-class Foo
+```php class Foo
```
Comment (gsteel) [inline docs/book/v2/inflector.md]:
```suggestion
```php
class Foo
```

**Review by gsteel — CHANGES_REQUESTED** [review]
Thanks @ramchale - I found only 1 problem 👍

**Comment by gsteel** [comment]
FYI, I drop this `Makefile` into my local Laminas repos so that you can easily build the docs website with `make docs` and open `./docs/html/index.html` in your browser

[Makefile.txt](https://github.com/user-attachments/files/17779111/Makefile.txt)

File: docs/book/v2/inflector.md (hunk 1 above)
Comment (ramchale) [reply docs/book/v2/inflector.md]: Awesome. Thanks
I've amended that

**Review by froschdesign — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**